docs/DEPLOYMENT.md

```diff
@@ -146,22 +146,68 @@ Follow the steps below to set up the staging environment on AWS.
        13. For auto-scaling, go with `Do not adjust the service's desired count`  for staging.
        14. Review and create the service.
 
-14. Next, go to your GitHub repository and create a new environment from the settings tab.
+11. Create and connect a `redis` cluster on AWS Elasticache to your backend app
+      1. Go to the ElastiCache dashboard on AWS. On the top right corner, select the region where you want to launch the Redis Cluster and click on **Create** button
+      2. Select **Redis** as your Cluster engine. In the checkbox below, choosing between *Cluster mode enabled* or *Cluster mode disabled* is your choice. Plio uses *Cluster mode disabled*. More details on cluster mode [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Replication.Redis-RedisCluster.html)
+      3. Choose **Amazon Cloud** as your location in the **Location** section
+      4. Fill all the details in the **Redis Details** section. Some pointers below:
+         1. Choose a name for your Redis Cluster, e.g. `plio-redis-staging`. Description is optional
+         2. For **Engine version compatibility**, it's recommended to choose the latest one.
```

Comment:
instead of recommended, we have used as "do this" vs "you can choose to do A or B"